mini-e2e.groovy

```diff
@@ -171,6 +171,12 @@ node('cico-workspace') {
 		}
 	}
 
+	catch (err) {
+		stage('log system status') {
+			ssh './system-status.sh'
```

Comment:
Just a question, is the script executable by default? Or is it safer to use `ssh 'sh system-status.sh'`?

Reply:
Not by default, but the permissions are stored in the git repository. You can check the permissions in the [patch format](https://github.com/ceph/ceph-csi/pull/2068.patch) of this PR, it contains the `0755` that indicates it is executable:
```
diff --git a/system-status.sh b/system-status.sh
new file mode 100755
```